frontend/src/components/me/config/UserConfig.jsx

```diff
@@ -0,0 +1,332 @@
+      "danger",
+      true
+    );
+    return false;
+  }
+  return true;
+}
+
+export function filterEmptyParams(plugins) {
+  return Object.keys(plugins)
+    .filter(
+      (key) =>
+        plugins[key].params && Object.keys(plugins[key].params).length > 0
+    )
+    .reduce((res, key) => Object.assign(res, { [key]: plugins[key] }), {});
+}
+
+export function Config({ configFilter, additionalConfigData }) {
+  const [
+    analyzers,
+    connectors,
+    retrieveAnalyzersConfiguration,
+    retrieveConnectorsConfiguration,
+  ] = usePluginConfigurationStore((state) => [
+    filterEmptyParams(state.analyzersJSON),
+    filterEmptyParams(state.connectorsJSON),
+    state.retrieveAnalyzersConfiguration,
+    state.retrieveConnectorsConfiguration,
+  ]);
+
+  const [respData, Loader, refetchConfig] = useAxiosComponentLoader(
+    {
+      url: CUSTOM_CONFIG_URI,
+    },
+    configFilter
+  );
+
+  const refetchAll = () => {
+    refetchConfig();
+    retrieveAnalyzersConfiguration();
+    retrieveConnectorsConfiguration();
+  };
+
+  return (
+    <Loader
+      render={() => (
+        <Formik initialValues={{ config: respData }} onSubmit={null}>
+          {({ values: configurations, setFieldValue }) => (
+            <Form>
+              <FieldArray name="config">
+                {({ remove, push }) => (
+                  <FormGroup row>
+                    <Col>
+                      {configurations.config && configurations.config.length > 0
+                        ? configurations.config.map((plugin, index) => {
+                            let plugins;
+                            let attributeList = [];
+                            let placeholder = "";
+                            if (plugin.type === "1") {
+                              plugins = analyzers;
+                            } else if (plugin.type === "2") {
+                              plugins = connectors;
+                            } else {
+                              plugins = {};
+                            }
+                            if (
+                              plugin.plugin_name &&
+                              plugins[plugin.plugin_name]
+                            )
+                              attributeList = Object.keys(
+                                plugins[plugin.plugin_name].params
+                              );
+                            if (
+                              attributeList.length > 0 &&
+                              plugin.attribute &&
+                              plugins[plugin.plugin_name].params[
+                                plugin.attribute
+                              ]
+                            ) {
+                              const { type } =
+                                plugins[plugin.plugin_name].params[
+                                  plugin.attribute
+                                ];
+                              if (type === "str") placeholder = '"string"';
```

Comment:
Removing the `"` would lead to inconsistency. For instance, in a list, you'll have to write `["5", 5]` to differentiate between strings. Hence in some places, strings would have a `"`, whereas it won't in other places.

Moreover, if we support a generic `JSON` type param in the future, we'll need to use the `"` for sure.

> I think you should check in the backend the type (same as int).

I'm checking all types in the backend but also ensuring proper JSON formatting for all of them.